lib/validate_encrypt_utils.pm

```diff
@@ -63,13 +117,36 @@ sub verify_number_of_encrypted_devices {
           Dumper($actual_number));
 }
 
+=head2 verify_cryptsetup_message
+
+  verify_cryptsetup_message($expected_message, $actual_message);
+
+=over 12
+=item C<$expected_message>
+  A string with the expected status message of a device
+=item C<$actual_message>
+  A string with the actual status message of a device
+=back
+=cut
 sub verify_cryptsetup_message {
     my ($expected_message, $actual_message) = @_;
     record_info("Assert volumes status", "Verify crypted volume status based on test_data expectations");
     assert_matches(qr/$expected_message/, $actual_message,
         "Message of cryptsetup status does not match regex");
 }
 
+=head2 verify_cryptsetup_properties
+
+  verify_cryptsetup_properties($expected_properties, $actual_properties);
+
+=over 12
+=item=over 12
```

Comment:
Minor thing, but do we need custom indent level? If so, could you make it consistent? As of now some have it, some have only `=over`. And do we need both? Tbh, I'm not sure what `=item=over 12` will do. Do you have perlpod generated for this file somewhere?

Reply:
i am not sure either but i removed it. I run also some validations and i fixed all i found.